clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -6845,3 +6847,57 @@ ExprResult SemaSYCL::ActOnUniqueStableNameExpr(SourceLocation OpLoc,
 
   return BuildUniqueStableNameExpr(OpLoc, LParen, RParen, TSI);
 }
+
+void SemaSYCL::performSYCLDelayedAttributesAnalaysis(const FunctionDecl *FD) {
+  if (UserProvidedSYCLKernelFunctions.contains(FD))
+    return;
+
+  auto isDependent =
+      [](const auto *Attr,
+         const std::initializer_list<const ConstantExpr *> &Exprs) {
+        return std::any_of(
+            Exprs.begin(), Exprs.end(), [](const ConstantExpr *CE) {
+              return CE && (CE->isValueDependent() || CE->isTypeDependent());
```

Comment:
Can you remove unused Attr parameter from the lambda?

Reply:
Sure